secret_number.js

```diff
@@ -1,4 +1,14 @@
 'use strict';
 module.exports = function() {
-  
+  let randomNumber = 0;
```

Comment:
so the line `module.exports = function() {` creates the start of your closure for you. At this point, the `randomNumber` variable is already private. You can skip the if conditional below by just setting `randomNumber` to a random number in the declaration.